Jint.Tests/Runtime/ExtensionMethods/FlurlExtensionTest.cs

```diff
@@ -0,0 +1,33 @@
+using Flurl.Http;
+
+namespace Jint.Tests.Runtime.ExtensionMethods
+{
+    public class FlurlExtensionTest
+    {
+        [RunnableInDebugOnlyAttribute]
+        public void CanUseFlurlExtensionMethods()
```

Comment:
Flaky test (networking), better not have it as long as the same logic is already tested somewhere lese.

Reply:
That's why iIt has the `RunnableInDebugOnlyAttribute`, it only runs under debugger.